scene/gui/tabs.cpp

```diff
@@ -630,6 +630,7 @@ int Tabs::get_tab_idx_at_point(const Point2 &p_point) const {
 
 void Tabs::set_tab_align(TabAlign p_align) {
 
+	ERR_FAIL_INDEX(p_align, 3);
```

Comment:
I actually used `*_MAX` earlier, then commit-amend'd back to 3. 😛
https://github.com/godotengine/godot/blob/master/scene/gui/label.cpp#L504
I was afraid that it'd affect performance, but I suppose it make sense to use `*_MAX`.